features-json/web-app-manifest.json

```
@@ -468,27 +467,27 @@
       "10.0-10.2":"n",
       "10.3":"n",
       "11.0-11.2":"n",
-      "11.3-11.4":"a #2",
-      "12.0-12.1":"a #2",
-      "12.2-12.5":"a #2",
-      "13.0-13.1":"a #2",
-      "13.2":"a #2",
-      "13.3":"a #2",
-      "13.4-13.7":"a #2",
-      "14.0-14.4":"a #2",
-      "14.5-14.8":"a #2",
-      "15.0-15.1":"a #2",
-      "15.2-15.3":"a #2",
-      "15.4":"a #2",
-      "15.5":"a #2",
-      "15.6":"a #2",
-      "16.0":"a #2",
-      "16.1":"a #2",
-      "16.2":"a #2",
-      "16.3":"a #2",
-      "16.4":"a",
-      "16.5":"a",
-      "17":"a"
+      "11.3-11.4":"y #2",
+      "12.0-12.1":"y #2",
+      "12.2-12.5":"y #2",
+      "13.0-13.1":"y #2",
+      "13.2":"y #2",
+      "13.3":"y #2",
+      "13.4-13.7":"y #2",
+      "14.0-14.4":"y #2",
+      "14.5-14.8":"y #2",
+      "15.0-15.1":"y #2",
+      "15.2-15.3":"y #2",
+      "15.4":"y #2",
+      "15.5":"y #2",
+      "15.6":"y #2",
+      "16.0":"y #2",
+      "16.1":"y #2",
+      "16.2":"y #2",
+      "16.3":"y #2",
```

With the current state I disagree with Safari on iOS before 16.4 getting `y`, as note 2 was the - imho valid - reason for partial.

```suggestion
      "11.3-11.4":"a #2",
      "12.0-12.1":"a #2",
      "12.2-12.5":"a #2",
      "13.0-13.1":"a #2",
      "13.2":"a #2",
      "13.3":"a #2",
      "13.4-13.7":"a #2",
      "14.0-14.4":"a #2",
      "14.5-14.8":"a #2",
      "15.0-15.1":"a #2",
      "15.2-15.3":"a #2",
      "15.4":"a #2",
      "15.5":"a #2",
      "15.6":"a #2",
      "16.0":"a #2",
      "16.1":"a #2",
      "16.2":"a #2",
      "16.3":"a #2",
```

For reference:
https://github.com/Fyrd/caniuse/pull/6616
> > Added support for third-party browsers to offer Add to Home Screen from the Share menu.
>
> plus all over in https://webkit.org/blog/13878/web-push-for-web-apps-on-ios-and-ipados/, mentioning different manifest members
> → Preliminary for now I've simply removed the note from https://caniuse.com/web-app-manifest, but further research is needed!

Perhaps that research has arrived with this PR, changing 16.4 and onwards.

---

+1, this is a unique problem on iOS because third party browsers are blocked, but it essentially means that any feature not supported in webviews will demote iOS to partial support.